adala/skills/collection/label_studio.py

```diff
@@ -79,10 +80,11 @@ async def aapply(
                 instructions_template=self.instructions,
                 response_model=ResponseModel,
             )
-            ner_tag = self.has_ner_tag()
-            if ner_tag:
+            for ner_tag in self.ner_tags():
                 input_field_name = ner_tag.objects[0].value.lstrip('$')
                 output_field_name = ner_tag.name
                 quote_string_field_name = 'text'
-                output = extract_indices(pd.concat([input, output], axis=1), input_field_name, output_field_name, quote_string_field_name)
+                df = pd.concat([input, output], axis=1)
+                output = validate_output_format_for_ner_tag(df, input_field_name, output_field_name)
```

Comment:
nice, now we can take out this additional call to `validate_output_format_for_ner_tag`, but other than that lgtm 👍

Reply:
Still need it, I added the call to EntityExtraction.extract_indices not the standalone extract_indices